mitodl/ocw-studio
# Exclude courses which have the download button disabled from mirror drives

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/4473

### Description (What does it do?)
This excludes the sites in offline mass build which has `hide_download` button set for them.

### How can this be tested?
1. Switch `main` branch in `ocw-studio`
2. Create a site locally at `localhost:8043/sites` or use existing one
3. In metadata form for site, set `hide_download` to `True`
4. Trigger a mass_build: `docker-compose exec web ./manage.py mass_publish <live/draft> --filter <site-name>` and wait for the build to complete successfully.
5. Open `minio` console at `localhost:9001` and check course data in `ocw-content-offline-<live/draft>-local/courses/<site-name>` for your site.
6. Verify that data has been updated recently.
7. Switch to `umar/4473-exclude-courses-which-have-the-download-button-disabled-from-mirror-drives` in `ocw-studio`
8. Repeat steps `2 - 6` and data should not be updated for `offline` version sites.
